src/components/appsettings/ReposFolderPicker.vue

```diff
@@ -61,6 +64,13 @@ export default {
     }
   },
   computed: {
+    labelText: function () {
+      if (this.label) {
+        return this.label;
+      } else {
+        return !this.reposFolder ? 'Where do you store your repositories' : 'Where to store/find repositories:';
+      }
```

Comment:
Very weird to inject a ternary in the middle of an `if` block. In general if a function returns a default value, it should be outside the final else.
```suggestion
      } else if (!this.reposFolder) {
        return 'Where do you store your repositories?';
      }
      return 'Where to store/find repositories:';
```